testui/elements/testui_element.py

```diff
@@ -205,41 +210,40 @@ def visible_for(self, seconds=1):
                            f'for {seconds}s')
         return self
 
-    def wait_until_visible(self, seconds=10.0, log=True):
+    def wait_until_visible(self, seconds=10.0, log=True) -> "Elements":
         start = time.time()
-        err = None
-        err_text = 'not'
+
+        is_not = self.__is_not
+
+        # Executes the loop at least once in case `seconds` <= 0
+        entered_loop = False
+        while time.time() < start + seconds or not entered_loop:
```

Comment:
the loop still is quite far from perfect, before logging error for element not visible we still wait 0.2sec. suggestion:

```py
is_visible = self.is_visble(log=False)
while time.time() < start + seconds and not is_visible:
    time.sleep(0.2)
    is_visible = self.is_visble(log=False)
if is_visible:
    if log: put_log()
    return self

show_error()
```